docs/source/learn/core_notebooks/pymc_overview.ipynb

```diff
@@ -43,7 +43,7 @@
     "\n",
     "Running PyMC requires a relatively recent Python interpreter, preferrably version 3.8 or greater. A complete Python installation for Mac OSX, Linux and Windows can most easily be obtained by downloading and installing the free [`Anaconda Python Distribution`](https://store.continuum.io/cshop/anaconda/) by ContinuumIO or the open source [Miniforge](https://github.com/conda-forge/miniforge).\n",
     "\n",
-    "Once Python is installed, follow the  {ref}`platform-specific installation guides <installation>` on the PyMC GitHub repository.\n",
+    "Once Python is installed, follow the [insallation guide](https://www.pymc.io/projects/docs/en/latest/installation.html) on the PyMC documentation site.\n",
```

Comment:
I think that should continue to be a sphinx-cross reference, we should only remove the platform-specific note

Reply:
I edited the comment. I am on my mobile and it won't let me commit though, some refresh issue, probably buggy internetI am on the train. I removed the prural in guides as there are no longer multiple guides

Reply:
I had the same problems. Had to do the commit locally..